packages/db/src/async_operations.ts

```diff
@@ -0,0 +1,33 @@
+import { toSnake } from 'snake-camel';
+
+interface CamelCaseAsyncOperation {
+  id: string
+  description: string
+  operationType: string
+  status: string
+  output?: object
+  taskArn?: string
+  createdAt?: Date
+  updatedAt?: Date
+}
+
+interface SnakeCaseAsyncOperation {
```

Comment:
So the point of these types is to handle the "optional" properties when translating Dynamo source data to an object with snake case fields for saving to RDS?

Reply:
Correct, the existing types have a hard requirement for the timestamps to exist, which is not what we need when we are preparing the initial parallel write.